.gitattributes

```diff
@@ -0,0 +1,3 @@
+vendor/**                      linguist-vendored
+config/rbac/role.yaml          linguist-generated
+config/rbac/role_binding.yaml  linguist-generated
```

Comment:
After fix the annotations in the code, this files are generated by `controller-gen`, so they are marked as generated in the repository.